src/EditorFeatures/Core.Wpf/Microsoft.CodeAnalysis.EditorFeatures.Wpf.csproj

```diff
@@ -18,6 +18,12 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
+    <ProjectReference Include="..\..\Interactive\Host\Microsoft.CodeAnalysis.InteractiveHost.csproj">
+      <Aliases>InteractiveHost</Aliases>
```

Comment:
Yes, it's because of linked files.

---
In reply to: [261343413](https://github.com/dotnet/roslyn/pull/29069#discussion_r261343413) [](ancestors = 261343413)